src/main.js

```diff
@@ -27,19 +37,39 @@
     }
 
     attachNonPreventedListener(document, "click", (e) => {
+        // ignore ctrl / meta clicks (may be opening in new tab or window)
+        if (e.ctrlKey || e.metaKey) {
+            return;
+        }
+		
         const body = document.body;
-
         for (var el = e.target; el && el !== body; el = el.parentNode) {
             if (el.nodeName.toLowerCase() === "a") {
+                // ignore links with a .no-ajaxy class
+                if (el.classList.contains('no-ajaxy')) {
+                    break;
+                }
                 const targetUrl = el.href;
 
-                if (!el.target && targetUrl && targetUrl.indexOf("http") === 0) {
+                if (!el.target && targetUrl && targetUrl.startsWith("http")) {
                     const currentUrl = location.href;
+                    const currentSplit = currentUrl.split("#",2);
+                    const targetSplit = targetUrl.split("#",2);
 
-                    if (targetUrl.split("#")[0] !== currentUrl.split("#")[0]) {
+                    if (targetSplit[0] !== currentSplit[0]) {
```

Comment:
Not what’s going on here.

Reply:
Changed the split call to have a maximum of two indexes and I used that result instead so I could quickly compare the left side (same as what's there) and later the right hand side (which includes the remainder of the url past an anchor). Removed the call to split because that works already in the variable above, no need to duplicate work.